Snakefile

```diff
@@ -40,6 +40,7 @@ def clock_rate(w):
     }
     return rate[(w.lineage, w.segment)]
 
+
```

Comment:
Be careful with whitespace. If you use Atom you set it to automatically remove trailing whitespace. It's less confusing if everyone contributing to a project strips trailing whitespace.